### ISSUE TITLE
Problem installing plugin

### ISSUE THREAD

**Issue by marc3k18** [t0bst4r/matterbridge-home-assistant-addon#36]
Hello,

I trying to install zigbee2mqtt addon but im still having some error. Its only for testing purpose!

Thanka
![Screenshot_20240609-182558](https://github.com/t0bst4r/matterbridge-home-assistant-addon/assets/97817024/13da2a74-9ffb-4f0a-91f3-1f6a981a2ecc)

**Comment by t0bst4r** [comment]
Hi, can you provide some more details? Are you able to get the docker logs from the plugin page (tab next to configuration) ?

**Comment by t0bst4r** [comment]
To be honest, I am wondering what you are doing there. This addon is publishing the entities you already have in Home Assistant as a matter bridge to other matter clients (like Alexa). 
why do you need zigbee2mqtt in there? 
wouldn’t it be more useful to have zigbee2mqtt in home assistant, have the entities in there and publish them via HA to this plugin?

**Comment by marc3k18** [comment]
Hi,

I want to use the climate device also.


[15:38:05.112] [Matterbridge] Installing plugin matterbridge-zigbee2mqtt...
[15:38:05.118] [Matterbridge] Failed to start child process: spawn sudo ENOENT
[15:38:05.118] [Matterbridge] Error installing plugin matterbridge-zigbee2mqtt
[15:38:05.120] [Matterbridge] Child process stdio streams have closed
with code -2
[15:38:10.151] [Matterbridge] WebSocket client disconnected
[15:38:10.318] [Matterbridge] WebSocketServer client connected
[15:38:10.337] [Matterbridge] WebSocket received message => Connected
to WebSocket: ws://10.18.0.250:8284
[15:38:14.374] [Matterbridge] WebSocket client disconnected

**Comment by t0bst4r** [comment]
Okay umm, i see. You are using the home assistant addon and you are trying to add the zigbee matterbridge plugin using the webui of matterbridge. Am i right?

The docker image I am building for this addon does not include any other plugin than mine. Even when the UI of matterbridge allows **adding** plugins which are already installed, it does not allow **installing** plugins which are missing.

i am not sure if I should add those matterbridge plugins to the addon. I should rather add support for the entity types of home assistant which are missing.

Since I don’t have any climate devices, I’d need help for that. What attributes does a entity has? What service calls are needed in home assistant?

as long as not all entity types are supported, unwould be fine adding the zigbee plugin (without any support from my side) . But i don’t think it’s a good solution.

**Comment by marc3k18** [comment]
Yes you are right. I'm trying to put some zigbee baseboard heater on Matter!

There is the attribute:

hvac_modes:
  - heat
min_temp: 5
max_temp: 30
target_temp_step: 0.5
current_temperature: 21
temperature: 5
hvac_action: idle
friendly_name: Z2M - T - Chambre Garçons
supported_features: 385


The only service need is the set target temperature.  To put them at off,
their need to be set at 5 degres.

There is the model: https://www.zigbee2mqtt.io/devices/HT402.html

**Comment by t0bst4r** [comment]
1. Those are the attributes from home assistant right? So it is connected to HA? Just to make sure.
2. Can you post the detailed service call in yaml format of how you set the temperature in home assistant (e.g. in automations) ?
3. Could you provide a list of all service calls available for this kind of entity?
4. Could you provide the entity_id of one of those devices?

**Comment by marc3k18** [comment]
1. Yes. Those are connect directly to zigbee2mqtt.

2.
service: climate.set_temperature
data_template:
  temperature: 20
  entity_id: climate.z2m_t_chambre_garcons

3. It's the only service call available for this model. It's also returning
the energy mesurement but I dont need it.

4. climate.z2m_t_chambre_garcons

**Comment by t0bst4r** [comment]
Thank you. I’ll give it a try in the next couple of days and then come back to you to test it 🙂
